docs/TektonConfig.md

```diff
@@ -151,8 +151,11 @@ pruner:
 
 - `resources`: supported resources for auto prune are `taskrun` and `pipelinerun`
 - `keep`: maximum number of resources to keep while deleting removing
+- `keep-since`: retain the resources younger than the specified value in minutes
 - `schedule`: how often to clean up resources. User can understand the schedule syntax [here][schedule].
 
+**NOTE**: `keep` and `keep-since` are mutually exclusive. Specify only one of them.
```

Comment:
this is not the case anymore with latest releases

Reply:
I put it by seeing this piece of code: https://github.com/tektoncd/operator/blob/main/pkg/apis/operator/v1alpha1/tektonconfig_validation.go#L77-L81
and this documentation: https://docs.openshift.com/container-platform/4.10/cicd/pipelines/automatic-pruning-taskrun-pipelinerun.html.

Reply:
May be we need to update this in operator, but this has been done in CLI https://github.com/tektoncd/cli/pull/1533

Reply:
Ok, I will open an issue for this.

Reply:
#974